EDU-1419 | Temporal Cloud guide | Introduction & get started content updates (#2507)
temporalio/documentation: docs-src/cloud/configuration.md

---
id: configuration
title: Defaults, limits, and settings - Temporal Cloud
sidebar_label: Configuration
description: Learn more about Temporal Cloud defaults, limits, and configurable settings.
tags:
  - temporal cloud
  - defaults
  - limits
  - configuration
  - settings
---

> Temporal Cloud has many operational features, many of which that support customization.

Reviewer: ```suggestion
Temporal Cloud has many operational features, and many of them can be customized.
```
